docs/source/trainer.rst

```diff
@@ -157,6 +157,41 @@ Once you're done training, feel free to run the test set!
 
     trainer.test(test_dataloaders=test_dataloader)
 
+Saving test predictions is simple !
+
+.. code-block:: python
+
+    import pandas as pd
+
+    class MyModel(LightningModule):
```

Comment:
It might be better to create two separate examples showing each version, or just do bother calls within the same test step as an example

Reply:
Done !